cmd/erasure-metadata.go

```diff
@@ -626,10 +626,10 @@ func getInternalReplicationState(m map[string]string) ReplicationState {
 	for k, v := range m {
 		switch {
 		case equals(k, ReservedMetadataPrefixLower+ReplicationTimestamp):
-			tm, _ := time.Parse(http.TimeFormat, v)
+			tm, _ := time.Parse(time.RFC3339Nano, v)
```

Comment:
@poornas For backcompat - should we try both if one fails?

Reply:
yes, that would be better